pkg/types/interfacetests/chain_reader_interface_tests.go

```diff
@@ -60,6 +62,15 @@ func RunChainReaderInterfaceTests[T TestingT[T]](t T, tester ChainReaderInterfac
 
 func runChainReaderGetLatestValueInterfaceTests[T TestingT[T]](t T, tester ChainReaderInterfaceTester[T]) {
 	tests := []testcase[T]{
+		{
```

Comment:
Highlights of the file:
- Call `StartChainReader` and `CloseChainReader` on every test that needs it. 
- Add error case for each of `GetLatestValue`, `BatchGetLatestValues`, and `QueryKey` when calling without starting the service previously.